tests/lib/rules/dispatch-actions.js

```diff
@@ -0,0 +1,60 @@
+'use strict'
+
+const get = require('lodash/get')
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+module.exports = {
+  meta: {
+    docs: {
+      description: 'Ensure actions are dispatched in mapDispatchToProps',
+      category: 'redux',
+      recommended: false,
+    },
+    fixable: null,
+    schema: [],
+  },
+
+  create: context => {
+    return {
+      VariableDeclarator(node) {
+        if (get(node, 'id.name') !== 'mapDispatchToProps') return
+
+        const boundFunctions = node.init.properties.filter(x => {
+          return get(x, 'value.type') === 'ArrowFunctionExpression'
+        })
+
+        if (!boundFunctions || !boundFunctions.length) return
+
+        const functionsWithMultipleExpresisons = boundFunctions.filter(x => {
+          return get(x, 'value.body.body.length') > 1
+        })
+
+        if (
+          !functionsWithMultipleExpresisons ||
+          !functionsWithMultipleExpresisons.length
+        )
+          return
+
+        const problemExpressions = functionsWithMultipleExpresisons.filter(
+          x => {
+            const body = get(x, 'value.body.body')
+            if (!body) return false
+
+            return x.value.body.body.some(x => {
+              return get(x, 'expression.type') === 'CallExpression'
```

Comment:
Isn't this also going to flag the times where `dispatch(something())` is called?

Reply:
It won't because it wouldn't get to this line. To get here, the mapDispatchToProps should be an object, not a function that returns an object: 

```typescript
// We'll get to this point in this snippet because it's a plain object
const mapDispatchToProps = {
  action: () => {
    action1()
    action2()
  },
}
```

```typescript
// this won't get flagged since it's a function
const mapDispatchToProps = dispatch => ({
  action: () => dispatch(actionName()),
})
```

Also [adding a test case here](https://github.com/earnwithdrop/eslint-plugin-drop/pull/10/files#diff-0ff34773b5fdfce555a6ddff831fd63622f08a91b8a3bd90336496feb4010854R47)